src/wccp/WccpConfig.cc

```diff
@@ -179,47 +137,35 @@ Service_Type_Defaulted(wccp::ServiceGroup::Type type, int line)
 }
 
 ts::Errata::Message
-Service_Type_Invalid(ts::ConstBuffer const &text, int line)
+Service_Type_Invalid(const std::string &text, int line)
```

Comment:
Use `string_view` in these case. Otherwise, as with YAMLCPP, use of a `string_view` or literal string will require the construction of a `std::string`. The C++ stream operators handle `string_view` correctly.

Reply:
I'll note that `ts::string_view` and `ts::TextView` are direct descendants of `ts::ConstBuffer`.